src/status_im2/navigation/options.cljs

```diff
@@ -44,12 +45,14 @@
 
 (defn navbar
   ([dark?]
-   {:navigationBar {:backgroundColor (if (or dark? (colors/dark?)) colors/neutral-100 colors/white)}})
+   {:navigationBar {:backgroundColor (if (or dark? (quo.theme/get-theme))
+                                       colors/neutral-100
+                                       colors/white)}})
   ([] (navbar nil)))
 
 (defn statusbar
   ([dark?]
-   (let [style (if (or dark? (colors/dark?)) :light :dark)]
+   (let [style (if (or dark? (quo.theme/get-theme)) :light :dark)]
```

Comment:
uses of `(quo.theme/get-theme)` are temporary as I will address this properly in a separate pr. there's more pieces involved to change navigation code wrt to theming

Reply:
Shouldn't we change the condition to `(or dark? (= :dark (quo.theme/get-theme)))` since we are replacing `(colors/dark?)` with `(quo.theme/get-theme)`?

Reply:
yep, thanks! :)